server/utils/validation/clientValidation.js

```diff
@@ -0,0 +1,24 @@
+const yup = require('yup');
+
+const schema = yup.object().shape({
+  fullName: yup.string().trim().required(),
+  email: yup.string().email().trim(),
+  mobileNumber: yup.string().trim(),
+  password: yup
+    .string()
+    .matches(
+      /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/,
+      'Must Contain 8 Characters, One Uppercase, One Lowercase, One Number and one special case Character'
+    )
+    .required(),
+  passwordConfirmation: yup.mixed().oneOf([yup.ref('password')]),
+  bankAccounts: yup.number().required(),
```

Comment:
```suggestion
  bankAccounts: yup.object().required(),
```

Reply:
nice one :+1:, I forget to change it when I changed fake data for clients because in first it was just a number for the account without any details
@Asmaathabet